getting-started-expressjs/index.cjs

```diff
@@ -0,0 +1,37 @@
+const { Metaplex, keypairIdentity, mockStorage } = require("@metaplex-foundation/js");
+const { Connection, clusterApiUrl, PublicKey, Keypair } = require("@solana/web3.js");
+const express = require('express');
+
+const kp = require('./keypair.json');
+const app = express();
+
+const arr = Object.values(kp._keypair.secretKey);
+const secret = new Uint8Array(arr);
+const myKeyPair = Keypair.fromSecretKey(secret);
```

Comment:
Since we're going to use a variable that points to a file in the user's computer, it might be better to use the filesystem package instead. Here's an example:

```js
const keypairFile = fs.readFileSync('~/.config/solana/id.json');
const secretKey = Buffer.from(JSON.parse(keypairFile.toString()));
const keypair = Keypair.fromSecretKey(secretKey);
```

Reply:
Done. Please Let me know if it's okay now

Reply:
That's great thanks!